packages/main/src/NavigationMenu.ts

```diff
@@ -67,10 +67,17 @@ class NavigationMenu extends Menu {
 	}
 
 	_itemMouseOver(e: MouseEvent) {
+		let opener;
 		if (isDesktop()) {
 			// respect mouseover only on desktop
-			const opener = e.target as OpenerStandardListItem;
-			let item = opener.associatedItem;
+			opener = e.target as OpenerStandardListItem;
+
+			// If the opener is a <ui5-icon> inside a Navigation item, we need to get the Navigation Item
+			if (opener.tagName === "UI5-ICON" && opener.parentElement) {
```

Comment:
`tagName` is not guaranteed. Replace it with `opener?.hasAttribute("ui5-icon")`.
Read more here https://sap.github.io/ui5-webcomponents/docs/contributing/DoD/#scoping-friendlymultiple-framework-safe-code